libs/utils/src/lib/pipes/array-contains/array-contains.pipe.ts

```diff
@@ -0,0 +1,40 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/**
+ * A pipe that will search an array to see if on of its objects contains specific values on provided keys.
+ *
+ * Usage:
+ * value | arrayContains : ['prop1', 'prop2', ...]
+ *
+ * Examples:
+ * 	{{
+ * 		[
+ * 			{ title: 'This is the title', description: 'This is the description' },
+ * 			{ title: 'This is the title' }
+ * 		] | arrayContains: ['description']
+ * 	}}
+ * 	Output: true
+ */
+@Pipe({ name: 'arrayContains' })
+export class ArrayContainsPipe implements PipeTransform {
+	public transform(values: unknown[], checkProps: string[] = []): any {
+		if (
+			!Array.isArray(values) ||
+			values.length === 0 ||
+			!Array.isArray(checkProps) ||
+			checkProps.length === 0
+		) {
+			return false;
+		}
+
+		const results = values.filter((item: unknown) => {
```

Comment:
`return values.some((item: unknown) => checkProps.some(key => (item[key]!== null && item[key]!== undefined)));`

☝️ Dit is hoe je het kan doen met de some operator. Is beter optimized dan via een filter, en je hoeft ook niet constant een const te assignen.